Sources/Document/Operation/TreeSytleOperation.swift

```diff
@@ -95,7 +98,8 @@ class TreeStyleOperation: Operation {
                 from: change.from,
                 to: change.to,
                 fromPath: change.fromPath,
-                value: attributes
+                toPath: change.toPath,
+                value: values!
```

Comment:
@humdrum 
It would be better to revise the values! code since change.value can be nil

Reply:
I changed the value type to optional. Thanks.